datalake/src/main/java/com/sequenceiq/datalake/flow/create/handler/SdxValidationHandler.java

```diff
@@ -0,0 +1,87 @@
+package com.sequenceiq.datalake.flow.create.handler;
+
+import static com.sequenceiq.cloudbreak.common.exception.NotFoundException.notFound;
+
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.dyngr.exception.PollerException;
+import com.dyngr.exception.PollerStoppedException;
+import com.dyngr.exception.UserBreakException;
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.flow.create.event.SdxCreateFailedEvent;
+import com.sequenceiq.datalake.flow.create.event.SdxValidationRequest;
+import com.sequenceiq.datalake.flow.create.event.SdxValidationSuccessEvent;
+import com.sequenceiq.datalake.repository.SdxClusterRepository;
+import com.sequenceiq.datalake.service.sdx.EnvironmentService;
+import com.sequenceiq.datalake.service.sdx.SdxRecommendationService;
+import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+import reactor.bus.Event;
+
+@Component
+public class SdxValidationHandler extends ExceptionCatcherEventHandler<SdxValidationRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SdxValidationHandler.class);
+
+    @Value("${sdx.db.operation.duration_min:60}")
```

Comment:
Its not a new property, it is just refactored and moved from StorageValidationHandler to here. And I am pretty sure there are a tons of similar properties without defaults in application.yml.

Reply:
if it's moved then it can be fine, but we don't introduce any new property with defaults in the code if it's not an overkill (like  you would have to set it for every service in the app yaml while it's used only in one).